osc-server/src/main/java/org/osc/core/broker/service/persistence/VirtualSystemEntityMgr.java

```diff
@@ -185,7 +185,6 @@ public static synchronized Long generateUniqueTag(EntityManager em, VirtualSyste
     }
 
     public static boolean isProtectingWorkload(VirtualSystem vs) {
-        return !vs.getServiceFunctionChains().isEmpty() ||
-                CollectionUtils.emptyIfNull(vs.getDeploymentSpecs()).stream().anyMatch(ds -> DeploymentSpecEntityMgr.isProtectingWorkload(ds));
+        return CollectionUtils.emptyIfNull(vs.getDeploymentSpecs()).stream().anyMatch(ds -> DeploymentSpecEntityMgr.isProtectingWorkload(ds));
```

Comment:
getServiceFunctionChains().isEmpty() should not be removed.

Reply:
Its required to remove this check because getServiceFunctionChains().isEmpty() will not separate the SFC call and would not be able to throw the appropriate error message.